client/app/vps/backup-storage/vps-backup-storage.controller.js

```diff
@@ -1,70 +1,38 @@
 class VpsBackupStorageCtrl {
-    constructor ($stateParams, $translate, CloudMessage, VpsActionService, VpsService) {
+    constructor ($stateParams, ControllerHelper, VpsActionService, VpsService) {
         this.serviceName = $stateParams.serviceName;
-        this.$translate = $translate;
-        this.CloudMessage = CloudMessage;
+        this.ControllerHelper = ControllerHelper;
         this.VpsActionService = VpsActionService;
         this.VpsService = VpsService;
-
-        this.loaders = {
-            init: false,
-            information: false,
-            vps: false
-        };
-        this.backup = {};
-        this.vps = {};
-
     }
 
-    $onInit () {
-        this.loadInformation();
-        this.loadBackup();
+    initLoaders () {
```

Comment:
I personally found ControllerHelper bloated and a result of our misunderstanding of normal angular services. This change is change is a bit the same of what was previously there and introduce a new way to initialise, which is missing the point of using a framework.

Reply:
Tried to apply what have been made to make our life easier. Controller Helper allows us to win in lisibility in my opinion and it handles loaders.